cors.go

```diff
@@ -357,6 +357,15 @@ func (c *Cors) logf(format string, a ...interface{}) {
 	}
 }
 
+// check the Origin of a request. No origin at all is also allowed.
+func (c *Cors) OriginAllowed(r *http.Request) bool {
+	origin := r.Header.Get("Origin")
+	if origin == "" {
```

Comment:
You may or may not want this default.

Reply:
Why don't you let the isOriginAllowed decide?

Reply:
Good question :)
Removed.